docs/source/conf.py

```diff
@@ -323,7 +323,7 @@ def find_source():
             # /home/docs/checkouts/readthedocs.org/user_builds/pytorch_lightning/checkouts/
             #  devel/pytorch_lightning/utilities/cls_experiment.py#L26-L176
             path_top = os.path.abspath(os.path.join("..", "..", ".."))
```

Comment:
here may need one more
```suggestion
            path_top = os.path.abspath(os.path.join("..", "..", "..", ".."))
```

Reply:
And the one with src folder too